frontend/styles/Login.module.css

```
@@ -0,0 +1,75 @@
+.background {
+  padding: 0;
+  margin: 0;
+  height: 100vh;
+  width: auto;
+  background-color: #f2f2f2;
+}
+
+.login-form {
+  width: 500px;
+  height: auto;
+  margin: 5% auto;
+  display: flex;
+  flex-direction: column;
+  background-color: white;
+  border-radius: 1rem;
+  padding: 20px 20px 20px 20px;
+}
+
+.title {
+  font-size: 1.3rem;
+  padding: 0;
+  margin: 0;
+}
+
+.login {
+  -webkit-appearance: none;
+  width: 30%;
+  min-width: 100px;
+  border-radius: 24px;
+  text-align: center;
+  padding: 15px 40px;
+  margin-top: 5px;
+  color: #fff;
+  font-size: 14px;
+  font-weight: 500;
+  box-shadow: 0px 2px 6px -1px rgba(0, 0, 0, 0.13);
+  background: #559cad;
+  border: none;
+  transition: all 0.3s ease;
+  outline: 0;
+}
+
+.login:hover {
+  transform: translateY(-3px);
+  box-shadow: 0 2px 6px -1px rgba(rgb(182, 157, 230), 0.65);
+  cursor: pointer;
+}
+
+.span {
+  font-weight: bold;
+  color: red;
+  font-size: 0.8rem;
+}
+
+.error-handling {
+  color: red;
+  font-weight: bold;
+  font-size: 1rem;
+  display: flex;
+  justify-content: center;
+  margin: 0;
+  padding: 0;
+}
+
+.input {
+  height: 40px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  border: 2px solid grey;
+  border-radius: 5px;
+  padding-left: 10px;
```

It would be nice if there was padding on both sides:
```suggestion
  padding: 0 10px;
```